include/vinecopulib/bicop/implementation/gaussian.ipp

```diff
@@ -78,7 +77,7 @@ inline Eigen::VectorXd GaussianBicop::get_start_parameters(const double tau)
 inline Eigen::MatrixXd GaussianBicop::tau_to_parameters(const double &tau)
 {
     Eigen::VectorXd parameters = this->parameters_;
-    parameters(0) = sin(tau * boost::math::constants::pi<double>() / 2);
+    parameters(0) = sin(tau * constant::pi / 2);
```

Comment:
It's funny that cran does not complain about std::sin here no?

Reply:
CRAN is totally inconsistent. I have no clue what it explains about and what not ^^

Reply:
True, but in any case, should'nt we use std::sin?

Reply:
will do